ambry-named-mysql/src/integration-test/java/com/github/ambry/named/MySqlNamedBlobDbIntegrationTest.java

```diff
@@ -244,4 +245,11 @@ private void cleanup() throws SQLException {
       }
     }
   }
+
+  private boolean checkRecordsEqual(NamedBlobRecord record1, NamedBlobRecord record2) {
```

Comment:
Instead of return a boolean, you can use this method to run assertEqual for each field.
So if one field is mismatch, it can be printed from the error message.
For example:
  private void testDatavaultPrincipal(X509Certificate cert, DatavaultIdentityToken token, String expectedCertString,
      String expectedTokenString) {
    DatavaultPrincipal principal = new DatavaultPrincipal(cert, token);
    assertEquals("Unexpected cert", cert, principal.getPeerCertificate());
    assertEquals("Unexpected token", token, principal.getToken());
    assertEquals("Unexpected toString value",
        String.format("DatavaultPrincipal{peerCertificate=%s, token=%s}", expectedCertString, expectedTokenString),
        principal.toString());
  }

Reply:
Sounds good, done.